src/components/ha-dialog.ts

```diff
@@ -119,6 +125,8 @@ export class HaDialog extends DialogBase {
         margin-top: var(--dialog-surface-margin-top);
         min-height: var(--mdc-dialog-min-height, auto);
         border-radius: var(--ha-dialog-border-radius, 28px);
+        -webkit--filter: var(--ha-dialog-surface--filter, none);
+        -filter: var(--ha-dialog-surface--filter, none);
```

Comment:
Shouldn't this be a single dash here: --ha-dialog-surface--filter?

Reply:
Good point. I fixed this in my fork but for some reason not on this branch.

Reply:
So, it left with a double dash?